src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -870,8 +878,13 @@ def _merge_input_dicts_by_line(
         input_dict: Mapping[str, List[Mapping[str, Any]]],
     ) -> List[Mapping[str, Mapping[str, Any]]]:
         for input_key, list_of_one_input in input_dict.items():
-            if len(list_of_one_input) == 0:
-                raise EmptyInputListError(f"List from key '{input_key}' is empty.")
+            if not list_of_one_input:
+                raise InputMappingError(
+                    message_format="The input for flow is incorrect. Input from key '{input_key}' is one empty list. "
+                    "Which means we cannot generate one line for the flow run. "
```

Comment:
"Which" is not a complete sentence. We should merge this part with the previous one. Something like:
"Input from key '{input_key}' is an empty list, which means ..."

Reply:
Fixed.